src/main/java/com/synopsys/integration/alert/channel/jira/cloud/web/JiraCustomEndpoint.java

```diff
@@ -87,6 +87,7 @@ public JiraCustomEndpoint(JiraChannelKey jiraChannelKey, CustomEndpointManager c
             if (!jiraPluginInstalled) {
                 return Optional.of(responseFactory.createNotFoundResponse("Was not able to confirm Jira Cloud successfully installed the Jira Cloud plugin. Please verify the installation on you Jira Cloud server."));
             }
+            return Optional.empty();
```

Comment:
Is this change so we are consistent with the jira server change?

Reply:
Yes. Just to ensure that we don't forget to actually return the error message.